files/bootstrap.sh

```diff
@@ -112,6 +112,77 @@ function get_pause_container_account_for_region () {
     esac
 }
 
+# Gets resource to reserve in a given region
```

Comment:
I'm not understanding what the "in a given region" refers to

Reply:
Ok, I understand now the link in the PR description.  Can we add the following 3 pieces of text as comments:

How Kubernetes calculates `Allocatable`:
```
Allocatable = Capacity - Reserved - Eviction Threshold
```

Reservations for memory resources:
```
255 MiB of memory for machines with less than 1 GB of memory
25% of the first 4GB of memory
20% of the next 4GB of memory (up to 8GB)
10% of the next 8GB of memory (up to 16GB)
6% of the next 112GB of memory (up to 128GB)
2% of any memory above 128GB
```

Reservations for CPU resources:
```
6% of the first core
1% of the next core (up to 2 cores)
0.5% of the next 2 cores (up to 4 cores)
0.25% of any cores above 4 cores
```